workflow/workflow.go

```diff
@@ -0,0 +1,92 @@
+package workflow
+
+import (
+	"fmt"
+
+	"github.com/mesg-foundation/engine/event"
+	"github.com/mesg-foundation/engine/hash"
+	eventsdk "github.com/mesg-foundation/engine/sdk/event"
+	executionsdk "github.com/mesg-foundation/engine/sdk/execution"
+)
+
+// Workflow exposes functions of the workflow
+type Workflow struct {
+	event       *eventsdk.Event
+	eventStream *eventsdk.Listener
```

Comment:
eventStream is used only in start method so it can be removed here

Reply:
This is used to check that the workflow engine is not started already and we should keep it in order to close the listener when we stop the workflow engine (not yet implemented)